### DISCUSSION TITLE
Add CA Certificate to theia On Run

### DISCUSSION THREAD

**Discussion by Allwin-Backya** [eclipse-theia/theia#7897] [General]
Hi,
   I want to know how to add ca certificate to theia image while running the theia command.

**Comment by akosyakov** [comment]
Have you seen https://github.com/eclipse-theia/theia/issues/7106 and https://github.com/eclipse-theia/theia/pull/960?

**Comment by Allwin-Backya** [comment]
yes I have checked, I have added cert and certkey for ssl. I also need to add ca certificate. That is where the problem rises. Theia is not running when i add mutiple cert files.

**Reply by akosyakov** [reply]
cc @marcdumais-work @vince-fugnitto Maybe you can help.

**Reply by vince-fugnitto** [reply]
I've only used `cert` and `certkey` for ssl (https) connection, but haven't also used `ca` certificate simultaneously. It is possible that it is unsupported at the moment but maybe someone who knows a bit more can chime in.

**Comment by marcdumais-work** [comment]
@Allwin-Backya Unfortunately I think none of us are experts in this area. 

Have you seen our `theia-https` [example application](https://github.com/theia-ide/theia-apps/tree/master/theia-https-docker)? 

Maybe you could modify this example to add your CA cert in the image and add whatever commands you need to run on it in [ssl_theia.sh](https://github.com/theia-ide/theia-apps/blob/master/theia-https-docker/ssl_theia.sh)?
